gst/nnstreamer/tensor_query/nnstreamer_edge_internal.c

```diff
@@ -72,16 +421,69 @@ nns_edge_release_handle (nns_edge_h edge_h)
     return NNS_EDGE_ERROR_INVALID_PARAMETER;
   }
 
+  eh->is_server = is_server;
+  if (!is_server && 0 == eh->recv_port)
+    eh->recv_port = _get_available_port ();
+
+  /** Initialize server src data. */
+  eh->cancellable = g_cancellable_new ();
+  eh->listener = g_socket_listener_new ();
+  g_socket_listener_set_backlog (eh->listener, N_BACKLOG);
+
+  if (!_nns_edge_get_saddr (eh->recv_ip, eh->recv_port, eh->cancellable,
+          &saddr)) {
+    nns_edge_loge ("Failed to get socket address");
+    ret = -EADDRNOTAVAIL;
+    goto error;
+  }
+  if (!g_socket_listener_add_address (eh->listener, saddr,
+          G_SOCKET_TYPE_STREAM, G_SOCKET_PROTOCOL_TCP, NULL, NULL, &err)) {
+    nns_edge_loge ("Failed to add address: %s", err->message);
+    g_clear_error (&err);
+    ret = -EADDRNOTAVAIL;
```

Comment:
let's define edge error code for this. (Or use NNS_EDGE_ERROR_CONNECTION_FAILURE)

Reply:
Updated to use `NNS_EDGE_ERROR_CONNECTION_FAILURE`. Thanks.